src/utils/currentLanguage.ts

```diff
@@ -4,3 +4,7 @@ export function getCurrentLanguage(): undefined | string {
   const currentSiteLanguage = Obj.root()?.language()
   return currentSiteLanguage === null ? 'en' : currentSiteLanguage
 }
+
+export function acceptLanguageHeader() {
```

Comment:
* [ ] FYI it's a misnomer as we return an object, which indicates plural. The red flag is code like: `headers: acceptLanguageHeader()`. Ideas:
  * rename to sth. like `languageHeaders`
  * return only the value `getCurrentLanguage() ?? 'en'`